spec/compiler/codegen/class_var_spec.cr

```diff
@@ -542,7 +542,7 @@ describe "Codegen: class var" do
       )).to_i.should eq(1)
   end
 
-  it "codegens generic class class var" do
+  it "codegens generic class var" do
```

Comment:
This is a new heuristic and I'm still gathering data from it. I'm not sure about this instance.

Reply:
No, this is a (generic class) with a (class var).